src/foam/nanos/auth/DeletedAwareDAO.java

```diff
@@ -0,0 +1,110 @@
+package foam.nanos.auth;
+
+import foam.core.FObject;
+import foam.core.X;
+import foam.dao.*;
+import foam.mlang.order.Comparator;
+import foam.mlang.predicate.Predicate;
+
+/*
```

Comment:
If you comment this with /** instead of /* then the comments will be included in the javadocs.